### PR TITLE
v0.9.0

### PR DESCRIPTION
**Pull request by juliotrigo**
* Bump up version to `v0.9.0`
* Amend `CHANGELOG`
* Bump up `dev` requirement version
* Improve documentation

### CODE REVIEW

**Comment by juliotrigo** [comment]
Changes for the next release. @mattbennett Please review.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,12 +5,31 @@ Here you can see the full list of changes between sqlalchemy-filters
...
+* Add compatibility (no official support) with Python 2.7 (#23 which
+  addresses #18 thanks to @itdependsnetworks)
+* Add support for Python 3.7 (#25)
+* Add support (tests) for PostgreSQL (#28)
+* Add reStructuredText linter (#27)
+* Fix and improve documentation (#21 thanks to @daviskirk, #28)
+* Add ``MANIFEST.in`` file (#27)
+* Tox improvements (#27)
+* Travis CI: use docker, ``xenial`` distribution and Python 3.7 (#28)
+* Improve PyPI classifiers (#25)
```
Comment (mattbennett) [inline CHANGELOG.rst]: The last four here relate to developer tooling, which probably isn't helpful in the CHANGELOG.

Same thing applies to the reStructuredText linter. It's not a material change that users of the library will want to be aware of.

**Review by mattbennett — APPROVED** [review]
I would trim some of the items in the release notes, but otherwise LGTM 👍

File: CHANGELOG.rst (hunk 1 above)
Comment (juliotrigo) [reply CHANGELOG.rst]: Thanks for the suggestion. I've removed those items from the CHANGELOG.

**Pull request merged**